app/templates/flat-page.hbs

```diff
@@ -34,7 +34,9 @@
                       <i class="fa fa-envelope fa-lg"></i>
                     </a>
                   </li>
-                  <li class="stfh edit-item" style="display: list-item;"><a data-url={{absoluteURL}} class="stf " target="_blank" href="{{adminRoot}}/admin/flatpages/flatpage/{{flatPage.id}}">Edit This</a></li>
+                  {{#if isStaff}}
+                    <li class="stfh edit-item" style="display: list-item;"><a data-url={{absoluteURL}} class="stf " target="_blank" href="{{adminRoot}}/admin/flatpages/flatpage/{{flatPage.id}}">Edit This</a></li>
```

Comment:
why are we using an inline style anyway?

Reply:
That's just how it was when i copied it over from what it was in publisher. I've moved it into flatpages .scss